packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -286,7 +286,7 @@ abstract class Finder {
   }
 
   @override
-  String toString() => 'widget with $description';
+  String toString() => 'Finder for a widget with $description';
```

Comment:
given that it actually returns an Element, not a Widget, maybe `'Finder for $description'`?
also it's helpful to have toStrings be distinguishable from surrounding output, so maybe `'[Finder for $description]'`?